components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -21,74 +21,141 @@
                 </div>
-              </div>
-            </div>
-
-            <div v-else>
-              <div class="media">
-                <div class="media-left">
-                  <BasicImage
-                    custom-class="is-32x32"
-                    :src="props.option.image || '/placeholder.webp'" />
-                  <!-- <div class="preview-media-wrapper">
-                  <PreviewMediaResolver
-                    v-if="!props.option.image && props.option.animation_url"
-                    :src="props.option.animation_url"
-                    :metadata="props.option.metadata"
-                    :mimeType="props.option.type"
-                  />
-                  </div> -->
+                <nuxt-link
+                  :to="{
+                    name: routeOf('explore'),
+                    query: { ...$route.query, tab: 'COLLECTION' },
+                  }">
+                  <div>See All --></div>
+                </nuxt-link>
+              </b-tab-item>
+              <b-tab-item label="NFTs" value="NFTs">
+                <div
+                  v-for="(item, idx) in nftSuggestion"
+                  :key="item.id"
+                  :value="item"
+                  :class="`mb-2 link-item ${
+                    idx === selectedIndex ? 'selected-item' : ''
+                  }`"
+                  @click="gotoGalleryItem(item)">
+                  <div class="media">
+                    <div class="media-left">
+                      <BasicImage
+                        custom-class="is-64x64 round-image"
+                        :src="item.image || '/placeholder.webp'" />
+                    </div>
+                    <div class="media-content">
+                      <div
+                        class="is-flex is-flex-direction-row is-justify-content-space-between pt-2 pr-2">
+                        <span class="main-title name">{{ item.name }}</span>
+                        <span>{{ urlPrefix.toUpperCase() }}</span>
+                      </div>
+                      <div
+                        class="is-flex is-flex-direction-row is-justify-content-space-between pt-1 pr-2">
+                        <span class="name">{{ item.collection?.name }}</span>
+                        <span>
+                          Price:
+                          <Money
+                            v-if="item.price && parseInt(item.price)"
+                            :value="item.price"
+                            inline /><span v-else>None</span>
+                        </span>
+                      </div>
+                    </div>
+                  </div>
                 </div>
-                <div class="media-content">
-                  {{ props.option.name }}
+                <nuxt-link
+                  :to="{
+                    name: routeOf('explore'),
+                    query: { ...$route.query, tab: 'GALLERY' },
+                  }">
+                  <div>See All --></div>
```

Comment:
```suggestion
                  <div>See All --></div>
```
missing `See All` translation